roles/ldapserver/tasks/main.yml

```diff
@@ -39,3 +39,197 @@
   loop:
     - people
     - groups
+
+- name: Place TLS cert
+  ansible.builtin.copy:
+    src: "{{ ldap_tls_cert_file }}"
+    group: openldap
+    mode: preserve
+    dest: /etc/ldap/ldaps.pem
+    remote_src: false
+  when: ldap_tls_cert_file
+
+- name: Place TLS private key
+  ansible.builtin.copy:
+    src: "{{ ldap_tls_key_file }}"
+    group: openldap
+    mode: 0640
+    dest: /etc/ldap/ldaps.key
+    remote_src: false
+  when: ldap_tls_key_file
```

Comment:
Why is the when needed? I would personally rewrite this to either first check for ` is defined and ldap_tls_key_file` or just assume a test certificate is there

Reply:
i moved all the TLS stuff under a 
```
when:
   - ldap_tls_cert_file
   - ldap_tls_key_file
```

are you saying instead of just listing those,  i should explicitly call them with `is defined` and combine them?